packages/expo/cli/start/doctor/dependencies/__tests__/validateDependenciesVersions-test.ts

```diff
@@ -0,0 +1,121 @@
+import { vol } from 'memfs';
+import path from 'path';
+import resolveFrom from 'resolve-from';
+
+import * as Log from '../../../../log';
+import { validateDependenciesVersionsAsync } from '../validateDependenciesVersions';
+
+const asMock = <T extends (...args: any[]) => any>(fn: T): jest.MockedFunction<T> =>
+  fn as jest.MockedFunction<T>;
+
+jest.mock(`../../../../log`);
+jest.mock('../bundledNativeModules', () => ({
+  getBundledNativeModulesAsync: () => ({
+    'expo-splash-screen': '~1.2.3',
+    'expo-updates': '~2.3.4',
+    firebase: '9.1.0',
+  }),
+}));
+
+describe(validateDependenciesVersionsAsync, () => {
+  const projectRoot = '/test-project';
+
+  beforeEach(() => {
+    vol.reset();
+  });
+
+  it('resolves to true when the installed packages match bundled native modules', async () => {
+    vol.fromJSON(
+      {
+        'node_modules/expo-splash-screen/package.json': JSON.stringify({
+          version: '1.2.3',
+        }),
+        'node_modules/expo-updates/package.json': JSON.stringify({
+          version: '2.3.4',
+        }),
+      },
+      projectRoot
+    );
+    const exp = {
+      sdkVersion: '41.0.0',
+    };
+    const pkg = {
+      dependencies: { 'expo-splash-screen': '~1.2.3', 'expo-updates': '~2.3.4' },
+    };
+
+    await expect(validateDependenciesVersionsAsync(projectRoot, exp as any, pkg)).resolves.toBe(
+      true
+    );
+  });
+
+  it('resolves to false when the installed packages do not match bundled native modules', async () => {
+    asMock(Log.warn).mockReset();
+    vol.fromJSON(
+      {
+        'node_modules/expo-splash-screen/package.json': JSON.stringify({
+          version: '0.2.3',
+        }),
+        'node_modules/expo-updates/package.json': JSON.stringify({
+          version: '1.3.4',
+        }),
+      },
+      projectRoot
+    );
+    const exp = {
+      sdkVersion: '41.0.0',
+    };
+    const pkg = {
+      dependencies: { 'expo-splash-screen': '~0.2.3', 'expo-updates': '~1.3.4' },
+    };
+
+    await expect(validateDependenciesVersionsAsync(projectRoot, exp as any, pkg)).resolves.toBe(
+      false
+    );
```

Comment:
```suggestion
    await expect(validateDependenciesVersionsAsync(projectRoot, exp as any, pkg)).resolves.toBeFalsy();
```

Reply:
can only be `false` doesn't need to account for an empty string, etc.

Reply:
It's just that test reporting is better with `toBeTruthy` and `toBeFalsy` than with `toBe(...)`.